File: bee_3170.rb
def main
    qntBolinhas = gets.to_i
    qntGalhos = gets.to_i

    bolasPrecisa = (qntGalhos / 2).to_i;
    bolasFaltam = bolasPrecisa - qntBolinhas;

    if bolasFaltam > 0
        puts "Faltam #{bolasFaltam} bolinha(s)";
    else
        puts "Amelia tem todas bolinhas!";
    end
end

main

File: bee_3170.py
# https://judge.beecrowd.com/pt/problems/view/3170

import math

def main():
    qntBolinhas = int(input())
    qntGalhos = int(input())

    bolasPrecisa = math.floor(qntGalhos / 2)
    bolasFaltam = bolasPrecisa - qntBolinhas

    if bolasFaltam > 0:
        print(f"Faltam {bolasFaltam} bolinha(s)")
    else:
        print(f"Amelia tem todas bolinhas!")

main()